internal/runners/activate/namespace.go

```diff
@@ -19,118 +16,78 @@ import (
-	if len(paths) > 0 {
-		targetPath, err := r.promptAvailablePaths(paths)
-		if err != nil {
-			return "", err
-		}
-		if targetPath != nil {
-			return *targetPath, nil
-		}
-	}
-
-	// Is targetPath STILL nil? Prompt the user for a path
-	userPath, err := r.promptForPathInput(namespace)
-	if err != nil {
+	// Validate that target path doesn't contain a config for a different namespace
+	if err := r.validatePath(namespace.Project, targetPath); err != nil {
 		return "", err
 	}
 
-	return userPath, nil
+	return targetPath, nil
 }
 
-func (r *NamespaceSelect) promptAvailablePaths(paths []string) (*string, error) {
-	if len(paths) == 0 {
-		return nil, nil
+func (r *NamespaceSelect) getProjectPath(namespace *project.Namespaced) (string, error) {
+	paths := projectfile.GetProjectPaths(r.config, namespace.String())
+	if len(paths) > 0 {
+		return paths[0], nil
 	}
 
-	noneStr := locale.T("activate_select_optout")
-	choices := append(paths, noneStr)
-	var defaultPath string
-	path, err := r.prompter.Select(locale.Tl("activate_existing_title", "Existing Checkout"), locale.T("activate_namespace_existing"), choices, &defaultPath)
+	targetPath, err := getSafeWorkDir()
 	if err != nil {
-		return nil, err
-	}
-	if path != "" && path != noneStr {
-		return &path, nil
+		return "", locale.NewError("err_get_wd", "Could not get safe working directory")
 	}
 
-	return nil, nil
+	return filepath.Join(targetPath, namespace.Project), nil
 }
 
-// promptForPathInput will prompt the user for a location to save the project at
-func (r *NamespaceSelect) promptForPathInput(namespace string) (string, error) {
-	wd, err := getSafeWorkDir()
-	if err != nil {
-		return "", errs.Wrap(err, "Runtime failure")
-	}
-
-	defDir := filepath.Join(wd, namespace)
-	directory, err := r.prompter.Input(
-		locale.Tl("choose_dest", "Choose Destination"),
-		locale.Tr("activate_namespace_location", namespace), &defDir, prompt.InputRequired)
+func (r *NamespaceSelect) validatePath(name string, path string) error {
+	empty, err := fileutils.IsEmptyDir(path)
 	if err != nil {
-		return "", err
+		return locale.WrapError(err, "err_namespace_empty_dir", "Could not verify if directory is empty")
 	}
-	if directory == "" {
-		return "", locale.NewError("err_must_provide_directory")
+	if empty {
+		return nil
 	}
 
-	logging.Debug("Using: %s", directory)
-
-	return directory, nil
-}
-
-func (r *NamespaceSelect) validatePath(namespace string, path string) error {
 	configFile := filepath.Join(path, constants.ConfigFileName)
 	if !fileutils.FileExists(configFile) {
-		return nil
+		// Directory is not empty and does not contain a config file
+		return errs.AddTips(
+			locale.NewError("err_directory_in_use", "Project directory at {{.V0}} is not empty. When activating a new project the project directory must be empty."),
+			locale.T("custom_path_tip", "To use a custom path when activating a project use the [ACTIONABLE]--path <path/to/project>[/RESET] flag"),
```

Comment:
```suggestion
			locale.NewError("err_directory_in_use", "Project directory at {{.V0}} is not empty. When activating a new project the project directory must be empty.\nTo use a custom path when activating a project use the [ACTIONABLE]--path <path/to/project>[/RESET] flag"),
			locale.T("custom_path_tip", "Use custom path with: [ACTIONABLE]--path <path/to/project>[/RESET] flag"),
```

Tips are meant to be short and concise. Note this suggestion is meant to convey the idea. I think this localization should move to the localization file as it's too long.